e2e/fixtures/authenticatedUserPages.ts

```diff
@@ -0,0 +1,141 @@
+import { test as base, Browser, expect, type Page } from '@playwright/test'
+import { unlinkSync } from 'fs'
+import { baseURL } from '../../playwright.config'
+import {
+  cleanupTestUser,
+  createTestUser,
+  getAdminPAT,
+  getUserIdFromLocalStorage,
+  loginTestUser,
+} from '../helpers/testUser'
+import {
+  storageStateDir,
+  userConfigs,
+  userPrefix,
+  UserPrefixes,
+  userValidatedPrefix,
+} from '../helpers/userConfig'
+import { waitForInitialPageLoad } from '../helpers/utils'
+
+type AuthenticatedUserPageTestFixtures = {
+  userPage: Page
+  validatedUserPage: Page
+}
+
+type StorageStatePaths = { [key in UserPrefixes]?: string }
+type AuthenticatedUserPageWorkerFixtures = {
+  createUsers: void
+  storageStatePaths: StorageStatePaths
+}
+
+const createPage = async (browser: Browser) => {
+  const page = await browser.newPage()
+  // specify baseURL so we call this method outside of a test
+  // ...i.e. where the baseURL test fixture is not available
+  await page.goto(baseURL)
```

Comment:
We use the page to call SRC methods during worker fixture cleanup, which is outside of a test.

Reply:
So if/when we can avoid the CJS/ESM issue and import an API client, we can get rid of this dependency on a `page` in the `testAuth` fixture, because we won't need to wait for the `window.SRC` object to be loaded?

Reply:
Yes, the fixtures that only make SRC API calls would be able to stop creating new pages, such as the `createUsers` worker fixture, which creates and deletes test users by making API calls via SRC's HttpClient.

However, we'd still need a `Page` to set up other fixtures -- for example, the new test fixtures (`userPage`, `validatedUserPage`) return a `Page` with a storage state for an authenticated user. Similarly, one of the new worker fixtures (`storageStatePaths`) uses a page to authenticate each user through the UI.

So `testAuth` (our [extension](https://playwright.dev/docs/api/class-test#test-extend) of `test`) would still need to create new `Page` objects, even after we can call the API from node.